### PR TITLE
[Doc] Fix doc issue of sendTransaction

### LINKED ISSUE #7185: docs - sendtransaction has wrong return in docs
https://docs.web3js.org/api/web3/class/Web3Eth#sendTransaction
sendTransaction will return a receipt object, not transactionHash
```
const transaction = {
  from: '0x6E599DA0bfF7A6598AC1224E4985430Bf16458a4',
  to: '0x6f1DF96865D09d21e8f3f9a7fbA3b17A11c7C53C',
  value: '0x1'
}

const transactionHash = await web3.eth.sendTransaction(transaction);
console.log(transactionHash);
> 0xdf7756865c2056ce34c4eabe4eff42ad251a9f920a1c620c00b4ea0988731d3f

web3.eth.sendTransaction(transaction).then(console.log);
> 0xdf7756865c2056ce34c4eabe4eff42ad251a9f920a1c620c00b4ea0988731d3f

web3.eth.sendTransaction(transaction).catch(console.log);
> <Some TransactionError>

// Example using options.ignoreGasPricing = true
web3.eth.sendTransaction(transaction, undefined, { ignoreGasPricing: true }).then(console.log);
> 0xdf7756865c2056ce34c4eabe4eff42ad251a9f920a1c620c00b4ea0988731d3f
```

#### Steps to reproduce the behavior

1. [First step]
2. [Second step]
3. [and so on...]

### PR DESCRIPTION
**Pull request by blackmoshui**
## Description

Fix #7185 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [x] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

**Review by avkos — APPROVED** [review]

**Comment by blackmoshui** [comment]
@luu-alex Hi, it seems that this PR does not cause the CI error. Could you please review the RP and help it?

**Comment by blackmoshui** [comment]
@mconnelly8 @luu-alex @avkos Could you help review this PR? 😄

File: packages/web3-eth/src/web3_eth.ts [hunk 1]
```diff
@@ -988,19 +988,61 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 	 *   value: '0x1'
 	 * }
 	 *
-	 * const transactionHash = await web3.eth.sendTransaction(transaction);
-	 * console.log(transactionHash);
-	 * > 0xdf7756865c2056ce34c4eabe4eff42ad251a9f920a1c620c00b4ea0988731d3f
+	 * const transactionReceipt = await web3.eth.sendTransaction(transaction);
```
Comment (jdevcs) [inline packages/web3-eth/src/web3_eth.ts:991]: For minor changes pls create issue so some one from web3 team can update it if required. Its for avoiding airdrop farming.

**Comment by blackmoshui** [comment]
@jdevcs I solved an issue that had existed for 2 months. I believe this issue was created by the maintainer team. I didn't see anyone expressing willingness to resolve it.

Small issues are the best choice for people who are just getting familiar with the project. Larger changes are not suitable for anyone who has just become familiar with the project.

This is an open-source project. If you don't want this project to be associated with airdrops, I suggest you inform all project parties not to give airdrops to contributors of web3.js. If you don't want open-source personnel to contribute code, you can directly set permissions.

Closing a PR for such reasons is unreasonable.

**Comment by jdevcs** [comment]
We appreciate your perspective and encourage you to submit code contributions on Github. You can also check out and help solve some bugs or add code contributions from the issues : https://github.com/web3/web3.js/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22 or from list of other issue https://github.com/web3/web3.js/issues. Your engagement is valuable to our project.

**Pull request closed without merging**